pkg/pdsutils/api/tenant.go

```diff
@@ -0,0 +1,44 @@
+// Package api comprises of all the components and associated CRUD functionality
+package api
+
+import (
+	"context"
+	status "net/http"
+
+	pds "github.com/portworx/pds-api-go-client/pds/v1alpha1"
+	"github.com/sirupsen/logrus"
+)
+
+// Tenant struct comprise of Context , PDS api client.
+type Tenant struct {
+	Context   context.Context
+	apiClient *pds.APIClient
+}
+
+// GetTenantsList return list of tenant objects.
+func (tenant *Tenant) GetTenantsList(accountID string) ([]pds.ModelsTenant, error) {
+	tenantClient := tenant.apiClient.TenantsApi
+	logrus.Info("Get list of Tenants.")
+	tenantsModel, res, err := tenantClient.ApiAccountsIdTenantsGet(tenant.Context, accountID).Execute()
+
+	if err != nil && res.StatusCode != status.StatusOK {
```

Comment:
in all the error check, no need to get the response status code. If the status code change from 200 to 202, the automation will fail, instead just rely on err != nil